testing.md

```diff
@@ -61,5 +61,71 @@ The following query is sufficient to test Native Query as it exercises mutliple
 | laptop | 3257.093 |
 | envelopes | 77.661 |
 
+# Manual Testing (for Direct Query)
 
+At this time, Direct Query must be tested semi-manually by reviewers, until such time as the Power Query
+SDK uses the same Mashup Engine as Power BI.
 
+Each Direct Query we want to run is stored in the `resources/direct_query` directory
+
+1. Ensure you have a local ADF running.
```

Comment:
It looks like the setup steps are repeated in all our tests sections. What about referencing the "setup" section in smoketest.md instead? This means only one place for updates, less risks of missing something (there is no mention of the environment variables that need to be set here for example).

Reply:
I removed smoketest.md and moved the text into testing.md. It didn't make sense to have two files.